py-polars/polars/config.py

```diff
@@ -1450,3 +1452,40 @@ def set_expr_depth_warning(cls, limit: int) -> type[Config]:
 
         os.environ["POLARS_MAX_EXPR_DEPTH"] = str(limit)
         return cls
+
+    @classmethod
+    def set_gpu_engine(cls, active: bool | None = None) -> type[Config]:
+        """
+        Set the default engine to use the GPU.
```

Comment:
*suggestion*: How about `set_default_engine(cls, engine: Literal["cpu", "gpu"])` instead.

That way, this interface is extensible to configuring (say) "streaming" as the default engine.